src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, 'Roboto', Arial, sans-serif should be 'Roboto', 'Arial', sans-serif, with Arial also in quotes as it is not a generic font family.